src/views/catalog/customize/drivers.ts

```diff
@@ -38,35 +38,33 @@ const getVmwareConfigMap = async (): Promise<any> => {
   throw lastException;
 };
 
-export const mountWinDriversToTemplate = async (template: V1Template) => {
-  let driversImage = DEFAULT_WINDOWS_DRIVERS_DISK_IMAGE;
+export const mountWinDriversToTemplate = async (template: V1Template): Promise<V1Template> => {
+  return await produce(template, async (draftTemplate) => {
+    let driversImage = DEFAULT_WINDOWS_DRIVERS_DISK_IMAGE;
 
-  try {
-    const configMap = await getVmwareConfigMap();
-    if (configMap?.data?.[VIRTIO_WIN_IMAGE]) driversImage = configMap.data[VIRTIO_WIN_IMAGE];
-  } catch (error) {
-    console.error(error);
-  }
+    try {
+      const configMap = await getVmwareConfigMap();
```

Comment:
`getVmwareConfigMap` we don't use VmWare any more it's virtio-win now,
you need to rename the method